src/controllers/sold.land.filter.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../utils/logger";
+import * as config from "../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.SOLD_LAND_FILTER_PAGE}`);
+
+    return res.render(config.SOLD_LAND_FILTER_PAGE, {
+      backLinkUrl: config.LANDING_URL,
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.SOLD_LAND_FILTER_PAGE}`);
```

Comment:
We need to store the information saved, it is not a blocker can be done in a separate story.

Reply:
It is work for another story :)